### ISSUE TITLE
AutoTrial stores data by name instead of by UUID

### ISSUE THREAD

**Issue by Pieter12345** [Pieter12345/Styms-Redstone-Server#16]
## Bug
If players change their name when they last logged out during a trial, they becomes stuck in the trial world with the trial rank, with no way to return to the normal world.

## Solution
- AutoTrial should store data by UUID instead of by name.
- Players with the trial rank in the trial world should automatically be 'fixed' if there is no trial data available for them (should never trigger, but would prevent players getting stuck due to bugs).

**Comment by Ecconia** [comment]
Automatically fixing the previous rank of a player is not trivial. They currently can be Regular or Learner, the trial system stores the previous rank. Without accessing the trial scripts data by name, its not possible to query the previous rank.

**Comment by Pieter12345** [comment]
Good point, this case indeed needs manual fixing and no automated action should be taken.
